README.md

```diff
@@ -10,7 +10,7 @@ This application is built on `React Native` and therefore assumes you have all t
 ### Prerequisites
 
 - The **default network configuration** is fetched from the `env/default.json` file, and as part of the first-time setup, you should provide these. Check the `env/default.json.example` file to see what keys are required.
-- Node version should be **12.1.0** or above.
+- Node version should be from **12.1.0** to **17.9.1**.
```

Comment:
Node 12 is really old.  I would increase the node version from 12.1.0 to 16.x if possible.
Even though node 16 will not be supported in a couple of weeks.
``from **16.x** to **17.9.1**.``